README.md

```diff
@@ -14,15 +14,25 @@ You need the ability to run docker and for it to be active in your session to ru
 ```
 docker ps -a
 ```
+If you are prompted with an error saying cannot connect to docker daemon, run the following command:
+
+```
+docker-machine start default
+```
 
 If this command doesn't return a TLS connection issue you are good to go otherwise the following command should connect Docker to your session,
 
 ```
 eval $(docker-machine env default) // your machine name may not be default use docker-machine ls to confirm
 ```
-
 Try the docker command above again, you should now see some form of output and not the original TLS error.
 
+If you need to stop and/or remove docker images, the commands are as follows
```

Comment:
Before this line add > Running this system causes containers to be created and ran via docker.